tests/engine/test_process_function.py

```diff
@@ -520,6 +520,50 @@ def test_input_serialization(argument, node_cls):
         assert result.value == argument
 
 
+# yapf: disable
+@pytest.mark.parametrize('default, node_cls', (
+    (True, orm.Bool),
+    ({'a': 1}, orm.Dict),
+    (1.0, orm.Float),
+    (1, orm.Int),
+    ('string', orm.Str),
+    ([1], orm.List),
+    (DummyEnum.VALUE, orm.EnumData),
+))
+# yapf: enable
+def test_default_serialization(default, node_cls):
+    """Test that Python base type default are automatically serialized to the AiiDA node counterpart."""
```

Comment:
```suggestion
    """Test that Python base type default is automatically serialized to the AiiDA node counterpart."""
```

Reply:
I will correct this like your other suggestion and make `default` plural instead.